build/linux/installer/conf/telegraf-rs.conf

```diff
@@ -124,26 +124,6 @@
   namedrop = ["agent_telemetry", "file"]
   #tagdrop = ["AgentVersion","AKS_RESOURCE_ID", "ACS_RESOURCE_NAME", "Region","ClusterName","ClusterType", "Computer", "ControllerType"]
 
-[[outputs.application_insights]]
```

Comment:
curios, are these unused plugins causing the cpu spike?

Reply:
No. These are generating logs and reported by Nuance.